src/audio/pipeline/pipeline-schedule.c

```diff
@@ -370,6 +370,9 @@ void pipeline_schedule_copy(struct pipeline *p, uint64_t start)
 	if (!pipeline_is_timer_driven(p))
 		sa_set_panic_on_delay(false);
 
+	if (task_is_active(p->pipe_task))
```

Comment:
Can we have a comment saying why we do this check.

Reply:
@lgirdwood done. BTW, this also *might* help with some IPC timeouts, currently in some cases the DAI pipeline can be scheduled before the host pipeline for playback, in which case of course the DAI pipeline has to skip a scheduling tick.